core/src/main/java/com/novoda/landingstrip/FastForwarder.java

```diff
@@ -0,0 +1,61 @@
+package com.novoda.landingstrip;
+
+import android.support.design.widget.Exposer;
+
+class FastForwarder {
+
+    static final int BYPASSED = -1;
```

Comment:
it's the value we set on the `State.getFastForwardPosition()` to bypass fast forwarding